docs/src/administration/web/configure-project.md

```diff
@@ -9,10 +9,26 @@ description: |
 
 ## General
 
-From the first page of the project settings, `General`, you can update the project name, or navigate to other project settings options on the left side of the screen.
+From the first page of the project settings, `General`, you can update the project name, set the timezone for the project or navigate to other project settings options on the left side of the screen.
 
 ![configure project](/images/management-console/settings-basics.png "0.7")
 
+The project timezone affects:
+
+- Automated backups.
+- Maintenance windows.
```

Comment:
These aren't on the same level. The timezone directly affects backups (which should get a link to the correct page). Maintenance windows are something you have to set up yourself and you can use the project timezone for that, if you want. But it's not the same, so I wouldn't use a list like this. Each should have more context.

Also, punctuation: https://developers.google.com/style/lists?hl=en#numbered-lettered-bulleted-lists